install/parse-cli.sh

```diff
@@ -22,7 +28,8 @@ SKIP_COMMIT_CHECK="${SKIP_COMMIT_CHECK:-}"
 while (( $# )); do
   case "$1" in
     -h | --help) show_help; exit;;
-    --no-user-prompt) SKIP_USER_PROMPT=1;;
+    --no-user-prompt) SKIP_USER_PROMPT=1;;  # deprecated
+    --skip-user-prompt) SKIP_USER_PROMPT=1;;
```

Comment:
We shouldn't have `--no-*` for one and `--skip-*` for the other. Let's standardize on `--skip`.